File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--updating-a-chart-in-a-worksheet-using-radio-buttons_5.cs
        private void lineChart_CheckedChanged(object sender, EventArgs e)
        {
            if (((RadioButton)sender).Checked)
            {
                this.selectedType = Microsoft.Office.Interop.Excel.XlChartType.xlLineMarkers;
                if (this.SelectionChanged != null)
                {
                    this.SelectionChanged(this, EventArgs.Empty);
                }
            }
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--updating-a-chart-in-a-worksheet-using-radio-buttons_5.vb
    Private Sub lineChart_CheckedChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) _
        Handles lineChart.CheckedChanged

        If (CType(sender, RadioButton).Checked) Then

            Me.selectedType = Microsoft.Office.Interop.Excel.XlChartType.xlLineMarkers
            RaiseEvent SelectionChanged(Me, EventArgs.Empty)
        End If
    End Sub